service/mantle/fedex/FedexServices.xml

```diff
@@ -0,0 +1,159 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<services xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/service-definition-2.1.xsd">
+
+    <service verb="validate" noun="PostalAddress">
+        <in-parameters>
```

Comment:
Please check the use of interface, we should implement the existing validate ShippingPostalAddress interface and add our custom validate service rather than adding the parameters separately.